recipes/ipycytoscape/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ipycytoscape" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95e41ec5c9d3e5633a5364eee891b3a0158338a6203d7d89a3d5768df61d78c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - networkx
+    - matplotlib-base
+    - ipywidgets >=7.5.0,<8.0.0
+    - traittypes >=0.2.1,<0.3.0
```

Comment:
Just to double-check
Your metadata does not contain information about those packages, just 
```
    - ipywidgets >=7.0.0
    - networkx
```
and it would be nice to add the python constrains to the metadata as well :)

Reply:
What do you mean by the metadata does not contain information about those packages?

Reply:
If you try to recover the metadata from pypi or from your setup it will not contain those packages as dependency, just those which I mentioned
Please see
https://pypi.org/pypi/ipycytoscape/json

and in the ``setup.py``
https://github.com/QuantStack/ipycytoscape/blob/master/setup.py#L88-L90

You can add a ``python_requires=">=3.0"`` to the ``setup.py`` and it will add that information to the metadata regarding the constrains about python

I just create a recipe using [grayskull](https://github.com/marcelotrevisani/grayskull/) and I was checking those things :smile:

Reply:
Ah ok, you mean setup.py.